sdks/builds/wasm.mk

```diff
@@ -57,8 +57,8 @@ endif
 WASM_RUNTIME_AC_VARS= \
 	ac_cv_func_shm_open_working_with_mmap=no
 
-WASM_RUNTIME_BASE_CFLAGS=-fexceptions $(if $(RELEASE),-Os -g,-O0 -ggdb3 -fno-omit-frame-pointer)
-WASM_RUNTIME_BASE_CXXFLAGS=$(WASM_RUNTIME_BASE_CFLAGS) -s DISABLE_EXCEPTION_CATCHING=0
+WASM_RUNTIME_BASE_CFLAGS=-fexceptions $(if $(RELEASE),-Os -g,-O0 -ggdb3 -fno-omit-frame-pointer) -s WASM_OBJECT_FILES=0
+WASM_RUNTIME_BASE_CXXFLAGS=$(WASM_RUNTIME_BASE_CFLAGS) -s DISABLE_EXCEPTION_CATCHING=0 -s WASM_OBJECT_FILES=0
```

Comment:
This should be done by adding a new runtime build, similarly to the runtime-threads build, preferably optional, controlled by a make define.

Reply:
I added the configuration as an opt-in, but should it be an opt-out like the rest of the configuration ? I'm confused by the pthreads define name.